### DISCUSSION TITLE
Bug - double display of the "Breadcrumb" component when switching to a nested directory

### DISCUSSION THREAD

**Discussion by shlyaiger** [community/community#64155] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

For two weeks, I noticed such an error in the user interface that when I go to the nested directory, the current directory + file is displayed on top of my code file twice[](url)
![Screenshot from 2023-08-18 10-43-14](https://github.com/community/community/assets/85574073/98736054-7183-45c1-bd78-14345841372a)

If you need additional information when solving this problem, write here.I will try to provide the data under which this bug was received

**Comment by AdamShwert** [comment, marked as answer]
You need to update your browser so that `container` CSS queries work!
